docs/api/params-for-successful-requests.md

```diff
@@ -0,0 +1,50 @@
+# Parameters for successful requests
+
+This section highlights the required and optional parameters for the To-Do service.
+
+To get a successful response, at a minimum, you must specify the properties listed under each task. 
+
+# Add a new task
+
+## URL: /tasks
+
+| Parameters| Type|Description|
+|---|---|---|
+|user_id | Integer   | Indicates the user-specified numeric value associated with a new task.|
+| title | Alphanumeric | Indicates a brief note about the task that you want to add.|
+| description | Varchar | Description of the task.   |
+| due_date  | Datetime   | Completion date of the task.   |
+| warning  | Varchar  | Indicates the user-specified note that can impact a task.|
```

Comment:
REST APIs typically use [JSON values](https://restfulapi.net/json-data-types/). These look like SQL data types, which, if required, would need to be expressed in a JSON data type, with whatever restrictions necessary to make them compatible. e.g. a DATETIME SQL type, would be stored in JSON as a string, formatted as date: YYYY-MM-... Date values in JSON are often expressed in [ISO 8601 format](https://www.iso.org/iso-8601-date-and-time-format.html).